wrapper/src/test/java/integration/refactored/container/tests/ReadWriteSplittingTests.java

```diff
@@ -368,21 +359,6 @@ public void test_setReadOnlyTrue_allReadersDown() throws SQLException {
     }
   }
 
-  @TestTemplate
```

Comment:
I propose that we remove this test. Most of the functionality it covers in the plugin is already covered by test_setReadOnlyTrue_allReadersDown. The part that is not covered in other tests is highly driver/database dependent (whether an exception is thrown or not depends on if  the underlying driver tries to communicate with the database when setReadOnly(true) is called). Let me know if you disagree